frontend/src/components/ViewTrips/trip-utils.js

```diff
@@ -92,3 +92,19 @@ export async function formatTripData(rawTripData) {
 
   return formattedTripObj;
 }
+
+/**
+ * Return collaborator emails corresponding to the collaborator uid's
+ * `collaboratorUidArr` in a comma separated string.
+ *
+ * @param {!string[]} collaboratorEmailArr Array of user emails sorted in
+ *     alphabetical order.
```

Comment:
Are you sorting these in alphabetical order somewhere else?

Reply:
I can't find where you're calling this function, so I guess I'm a bit confused by where you're doing the alphabetization.

Reply:
`moveCurUserEmailToFront` is called in `trip.js`. The collaborator emails are returned in alphabetical order from `AuthUtils` so this test just sets up a situation similar what will actually happen. But in terms of alphabetization, all I care about in this function is that the current user is moved to the front and the other elements in the array maintain the order otherwise. Does that help clarify? 

I will add a comment for this to try to clear up similar confusion in the future.